src/freertos_drivers/spiffs/SPIFFS.cxx

```diff
@@ -45,6 +46,20 @@
 #define O_DIRECT _FDIRECT
 #endif
 
+/// Provide mutex lock.
```

Comment:
All of the Doxygen comment header blocks for function headers should go in their respective header files, if applicable, and not in the the *.cxx file.  Keep them all in one place.

Reply:
all cleaned up in this file.